### PR TITLE
Admin UI 6 

### PR DESCRIPTION
**Pull request by marcosihe**
Forms view. 
Dependencies installed: material-UI
TypeScript was not applied yet

### CODE REVIEW

File: src/company.js [hunk 1]
```diff
@@ -17,41 +25,17 @@ export const CompanyList = () => (
...
-            <TextInput source="name" />
-            <TextInput source="address" />
-            <TextInput source="zipCode" />
-            <TextInput source="city" />
-            <TextInput source="state" />
-            <TextInput source="country" />
-        </SimpleForm>
-    </Create>
+    <CreateEntityLayout inputsList={inputsList} referenceValues={referenceValues} />
 );
```
Comment (agustinb4) [inline src/company.js:41]: Please add a new line at the end of the files

File: src/components/forms/CreateEntityLayout.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            <Grid item xs={12} sm={6} md={4}>
+              <ReferenceInputItem referenceValues={referenceValues} />
+            </Grid>
+          )}
+          <TextInputGroup list={inputsList} />
+          {birthDate && (
+            <Grid item xs={12} sm={6} md={4}>
+              <DateInput source="birthDate" fullWidth />
+            </Grid>
+          )}
```
Comment (agustinb4) [inline src/components/forms/CreateEntityLayout.js:22]: You could remove `birthDate` param and add to the input list the field type. Then in TextInputGroup ask about field type and render a DateInput or a TextInput

File: src/projects.js [hunk 3]
```diff
@@ -35,35 +33,12 @@ export const ProjectList = () => (
...
-            </ReferenceInput>
-            <ReferenceInput source="projectTypeId" reference="project-types">
-                <SelectInput source="name" />
-            </ReferenceInput>
-            <TextInput source="name" />
-            <DateInput source="startDate" />
-            <DateInput source="endDate" />
-            <RichTextInput source="notes" />
-        </SimpleForm>
+        <ProjectFormLayout referenceValuesC={referenceValuesC} referenceValuesP={referenceValuesP} />
```
Comment (agustinb4) [inline src/projects.js:36]: i think we could change referenceValuesC and referenceValuesP to better names, like projectReferences and clientReferences

**Review by agustinb4 — CHANGES_REQUESTED** [review]

File: src/company.js (hunk 1 above)
Comment (marcosihe) [reply src/company.js]: Done

File: src/components/forms/CreateEntityLayout.js (hunk 2 above)
Comment (marcosihe) [reply src/components/forms/CreateEntityLayout.js]: Done! I liked your approach, thanks for that!!

File: src/projects.js (hunk 3 above)
Comment (marcosihe) [reply src/projects.js]: Done!

**Review by agustinb4 — APPROVED** [review]
Looks good!

**Pull request merged**